lib/ui/utilities/validation.dart

```diff
@@ -1,7 +1,6 @@
 import 'package:receipe_app/ui/utilities/string_utils.dart';
 
 class Validation {
-
   static String? validateEmail(String? s) {
     if (StringUtil.isEmpty(s)) {
       return "Email cannot be empty";
```

Comment:
kindly help with this as part of the internalization

Reply:
I do not understand